apps/avatax/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
is it safe?

Reply:
I checked and Next.js supports it from at least 1 year: https://github.com/vercel/next.js/pull/51065. I also didn't observe AvaTax suspicious behaviour on PR while testing

Reply:
Also if it helps our payment apps have `bundler` set as well 😄